<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="SQA-CRS-021" name="prerate.prerateTestNGSlow" tests="17" failures="8" timestamp="13 Nov 2020 21:03:54 GMT" time="847.604" errors="3">
  <testcase name="testMethod1" time="46.727" classname="prerate.prerateTestNGSlow"/>
  <testcase name="testMethod1" time="66.471" classname="prerate.prerateTestNGSlow"/>
  <testcase name="testMethod1" time="68.255" classname="prerate.prerateTestNGSlow">
    <failure type="java.lang.AssertionError" message="Sum of Charge Details does not equal Payment Total Amount">
      <![CDATA[java.lang.AssertionError: Sum of Charge Details does not equal Payment Total Amount
	at org.testng.Assert.fail(Assert.java:93)
	at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1248)
	at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="0.141" classname="prerate.prerateTestNGSlow"/>
  <testcase name="testMethod2" time="0.000" classname="prerate.prerateTestNGSlow">
    <error type="org.testng.TestNGException" message="
Data Provider public synchronized java.lang.Object[][] prerate.prerateTestNGSlow.dataProviderMethod2() must return either Object[][] or Iterator&lt;Object&gt;[], not class [[Ljava.lang.Object;">
      <![CDATA[org.testng.TestNGException: 
Data Provider public synchronized java.lang.Object[][] prerate.prerateTestNGSlow.dataProviderMethod2() must return either Object[][] or Iterator<Object>[], not class [[Ljava.lang.Object;
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:509)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1308)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1036)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1126)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- testMethod2 -->
  <testcase name="testMethod1" time="37.863" classname="prerate.prerateTestNGSlow">
    <failure type="java.lang.AssertionError" message="Override Disabled.">
      <![CDATA[java.lang.AssertionError: Override Disabled.
	at org.testng.Assert.fail(Assert.java:93)
	at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1135)
	at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="47.555" classname="prerate.prerateTestNGSlow"/>
  <testcase name="testMethod3" time="0.000" classname="prerate.prerateTestNGSlow">
    <error type="org.testng.TestNGException" message="
Data Provider public synchronized java.lang.Object[][] prerate.prerateTestNGSlow.dataProviderMethod3() must return either Object[][] or Iterator&lt;Object&gt;[], not class [[Ljava.lang.Object;">
      <![CDATA[org.testng.TestNGException: 
Data Provider public synchronized java.lang.Object[][] prerate.prerateTestNGSlow.dataProviderMethod3() must return either Object[][] or Iterator<Object>[], not class [[Ljava.lang.Object;
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:509)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1308)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1036)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1126)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- testMethod3 -->
  <testcase name="testMethod1" time="107.135" classname="prerate.prerateTestNGSlow">
    <failure type="java.lang.AssertionError" message="Technical Error Occurred in Edits; Try again ,3310">
      <![CDATA[java.lang.AssertionError: Technical Error Occurred in Edits; Try again ,3310
	at org.testng.Assert.fail(Assert.java:93)
	at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1212)
	at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="58.868" classname="prerate.prerateTestNGSlow">
    <failure type="java.lang.AssertionError" message="Technical Error Occurred in Edits; Try again ,3310">
      <![CDATA[java.lang.AssertionError: Technical Error Occurred in Edits; Try again ,3310
	at org.testng.Assert.fail(Assert.java:93)
	at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1212)
	at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="0.124" classname="prerate.prerateTestNGSlow"/>
  <testcase name="testMethod4" time="0.000" classname="prerate.prerateTestNGSlow">
    <error type="org.testng.TestNGException" message="
Data Provider public synchronized java.lang.Object[][] prerate.prerateTestNGSlow.dataProviderMethod4() must return either Object[][] or Iterator&lt;Object&gt;[], not class [[Ljava.lang.Object;">
      <![CDATA[org.testng.TestNGException: 
Data Provider public synchronized java.lang.Object[][] prerate.prerateTestNGSlow.dataProviderMethod4() must return either Object[][] or Iterator<Object>[], not class [[Ljava.lang.Object;
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:509)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1308)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1036)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1126)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- testMethod4 -->
  <testcase name="testMethod1" time="82.397" classname="prerate.prerateTestNGSlow"/>
  <testcase name="testMethod1" time="58.077" classname="prerate.prerateTestNGSlow">
    <failure type="java.lang.AssertionError" message="Technical Error Occurred in Edits; Try again ,3310">
      <![CDATA[java.lang.AssertionError: Technical Error Occurred in Edits; Try again ,3310
	at org.testng.Assert.fail(Assert.java:93)
	at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1212)
	at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="58.552" classname="prerate.prerateTestNGSlow">
    <failure type="java.lang.AssertionError" message="Technical Error Occurred in Edits; Try again ,3310">
      <![CDATA[java.lang.AssertionError: Technical Error Occurred in Edits; Try again ,3310
	at org.testng.Assert.fail(Assert.java:93)
	at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1212)
	at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="158.287" classname="prerate.prerateTestNGSlow">
    <failure type="java.lang.AssertionError" message="Technical Error Occurred in Edits; Try again ,3310">
      <![CDATA[java.lang.AssertionError: Technical Error Occurred in Edits; Try again ,3310
	at org.testng.Assert.fail(Assert.java:93)
	at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1212)
	at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
  <testcase name="testMethod1" time="57.152" classname="prerate.prerateTestNGSlow">
    <failure type="java.lang.AssertionError" message="Technical Error Occurred in Edits; Try again ,6966,7169">
      <![CDATA[java.lang.AssertionError: Technical Error Occurred in Edits; Try again ,6966,7169
	at org.testng.Assert.fail(Assert.java:93)
	at prerate.prerateTestNGSlow.doPrerate(prerateTestNGSlow.java:1212)
	at prerate.prerateTestNGSlow.testMethod1(prerateTestNGSlow.java:540)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testMethod1 -->
</testsuite> <!-- prerate.prerateTestNGSlow -->
